src/controllers/ReportController.js

```diff
@@ -0,0 +1,71 @@
+import models from '../db/models';
+import Response from '../helpers/response';
```

Comment:
Kindly explain why it seems like you are working on both notification and report art. I am a little bit confused

Reply:
This must have occurred while merging conflicts from staging